openshift/4.0/templates/base-images/dotnet50/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.redhat.io/ubi8/dotnet-50
```

Comment:
Attempts to build dotnet 5.0 with Red Hat images.  So far not successful, but I'd like to solve this eventually.